### PR TITLE
add test for different work dirs with indexes

### PR DESCRIPTION
**Pull request by smacker**
Fix: #333

To test locally for now it's necessary to do `rm -rf ~/.srcd/gitbase/`. It will be magically fixed when #355 lands in master.

### CODE REVIEW

**Comment by smacker** [comment]
Tests are failing in travis but pass locally somehow. I'll investigate.

**Comment by smacker** [comment]
Rebased on master, updated checks of explain output.

File: cmd/srcd/cmd/sql_test.go [hunk 1]
```diff
@@ -211,6 +214,97 @@ func (s *SQLTestSuite) TestREPL() {
...
+	require.NoError(err)
+
+	cmd := exec.Command("git", "init", repoPath)
+	err = cmd.Run()
+	require.NoError(err)
+
+	_, err = s.RunInit(context.TODO(), s.testDir)
+	require.NoError(err)
+
+	buf, err = s.RunSQL(context.TODO(), "SHOW INDEX FROM repositories")
```
Comment (se7entyse7en) [inline cmd/srcd/cmd/sql_test.go]: We could reuse the `testQueryWithIndex` by changing its signature. Adding the `repo_id string` and `contains bool` should be enough to cover this case.

Comment (smacker) [reply cmd/srcd/cmd/sql_test.go]: I'm not sure if it makes the code simpler. But applied.

**Comment by smacker** [comment]
> 2 workdirs inside the tmp

means both of them will be empty. I prefer to have at least 1 workdir as real repository with commits. Just in case :)

**Pull request merged**